### ISSUE TITLE
A mistake in some of your jupyter notebooks!

### ISSUE THREAD

**Issue by dog14230pp** [cutz-j/T-GD#1]
Hello
I found that there might be a mistake in your jupyter notebook.
Such as 'efficientnetb0_128_pggan_transfer_star_aug_2000shot_self.ipynb'
I think the code: ` loss =  loss_main + 0.1*loss_sp + 0.1*loss_cls` is wrong.
And the correct code should be: ` loss =  loss_main + sp_alpha * loss_sp + sp_alpha * loss_cls` .
Maybe the mistake can appear in others' jupyter notebook, so please check others' notebook again.

**Comment by cutz-j** [comment]
Hello, thanks for interest of T-GD

Jupyter notebooks were uploaded in the middle of experiments (also not
updated in the repository), so they can be different in the paper or not
complete code.

transfer.py in Github is right (final).

        sp_gamma = 0
        sigmoid = nn.Sigmoid()
        sp_gamma += opt.s*sigmoid(-teacher_loss)
        
        outputs = student_model(inputs)
        loss_main = criterion(outputs, targets)
        loss_cls = 0
        loss_sp = 0
        loss_cls = reg_cls(student_model)
        loss_sp = reg_l2sp(student_model)
        loss =  loss_main + sp_gamma*loss_sp + sp_gamma*loss_cls

Sorry to disturb you.
Thanks. Regards

- 2020.8.16. Hyeonseong Jeon -

2020년 8월 16일 (일) 오후 7:11, Hao-Lun Sun <notifications@github.com>님이 작성:
